testing/tests/DevExpress.ui.widgets.editors/radioGroup.tests.js

```diff
@@ -135,6 +135,21 @@ module('buttons group rendering', () => {
         instance.option('dataSource', [1, 2, 3]);
         assert.strictEqual(onContentReadyHandler.callCount, 2);
     });
+
+    test('onContentReady - subscription using "on" method', function(assert) {
+        const onContentReadyHandler = sinon.stub();
+
+        const instance = getInstance(
+            createRadioGroup({
+                dataSource: ['str1', 'str2', 'str3']
+            })
+        );
+
+        instance.on('contentReady', onContentReadyHandler);
```

Comment:
I think it's worth checking that the radio buttons are rendered at the time of the event (check the markup in the handler).

Reply:
@DokaRus @ksercs 
So, do we need to make this test async?

Reply:
Yep. We can add `expect` assertion and resolve async promise on the `contentReady` event

Reply:
Is it ok now?